dev/dockerfiles/local-artifact-mirror/Dockerfile.ttlsh

```diff
@@ -22,6 +22,3 @@ FROM debian:bookworm-slim
 
 COPY --from=build /app/local-artifact-mirror/bin/local-artifact-mirror /usr/bin/local-artifact-mirror
 RUN ln -s /usr/bin/local-artifact-mirror /usr/local/bin/local-artifact-mirror
-
-RUN groupadd -r lam && useradd -r -u 1000 -g lam lam
-USER 1000
```

Comment:
this caused airgap upgrades in dev env to fail because cli commands need to run as root in the artifacts job